internal/dlc/execution.go

```diff
@@ -0,0 +1,184 @@
+package dlc
+
+import (
+	"errors"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/dgarage/dlc/internal/script"
+)
+
+// ContractExecutionTx constructs a contract execution tx (CET) using pubkeys and given condition.
+// Both parties have different transactions signed by the other side.
+//
+// input:
+//   [0]:fund transaction output[0]
+// output:
+//   [0]:settlement script
+//   [1]:p2wpkh (option)
```

Comment:
Are the input and outputs in the above comment correspond to `ContractExecutionTx` function's parameters and return structs?

Reply:
Those are txins/outs. I updated the comment.